activation/activation.go

```diff
@@ -154,7 +127,25 @@ func (b *Builder) CreatePoETChallenge(epoch types.EpochId) error {
 	if err != nil {
 		return err
 	}
-	return b.PublishActivationTx(npst)
+	//todo: check if view should be latest layer -1
+	atx := types.NewActivationTxWithcChallenge(challenge, b.activeSet.GetActiveSetSize(b.mesh.LatestLayerId()-1), b.mesh.GetLatestView(), npst)
+
+	buf, err := types.AtxAsBytes(atx)
+	if err != nil {
+		return err
+	}
+	//todo: should we do something about it? wait for something?
```

Comment:
Is this `todo` in the right place? Seems like it should be here 👇?
https://github.com/spacemeshos/go-spacemesh/pull/806/commits/daea588acd0b6571efef85ad644f7a583d169acf#diff-1ba4cf64f19ae66633cb8b7ca7158fabR103